doc/man7/ossl-guide-libssl-introduction.pod

```diff
@@ -0,0 +1,114 @@
+=pod
+
+=head1 NAME
+
+ossl-guide-libssl-introduction, ssl
+- OpenSSL Guide: An introduction to libssl
+
+=head1 INTRODUCTION
+
+The OpenSSL C<libssl> library provides implementations of several secure network
+communications protocols. Specifically it provides SSL/TLS (SSLv3, TLSv1,
+TLSv1.1, TLSv1.2 and TLSv1.3), DTLS (DTLSv1 and DTLSv1.2) and QUIC (client side
```

Comment:
Does it make sense to create a ticket to make an update here once we have server-side too?

Reply:
Done